arm-dns/2016-04-01/swagger/dns.json

```diff
@@ -937,6 +937,7 @@
         "TTL": {
           "type": "integer",
           "format": "int64",
+	   "x-ms-client-name": "ttl",
```

Comment:
From the source code [place1](https://github.com/Azure/autorest/blob/master/src/generator/AutoRest.CSharp/CSharpCodeNamer.cs#L167), [place2](https://github.com/Azure/autorest/blob/master/src/core/AutoRest.Core/CodeNamer.cs#L364), it looks that this is a no-op.